block-everything.txt

```diff
@@ -471,7 +476,13 @@ smashingmagazine.com##.c-promo-box--ad
 space.com##.post-gallery-item-ad
 spectator.us##.article_set_wrap:style(position: static !important;)
 spectator.us##.otherheader:style(position: absolute !important;)
+stackexchange.com##.tex2jax_ignore
```

Comment:
I don't mind the Hot Network Questions box. As long as it's not floating and not inline, I think it's OK for a website to have a "here's some other articles" box.